yarn.lock

```diff
@@ -531,10 +531,10 @@
     "@types/yargs" "^15.0.0"
     chalk "^4.0.0"
 
-"@nitric/api@v0.19.0-rc.2":
-  version "0.19.0-rc.2"
-  resolved "https://registry.yarnpkg.com/@nitric/api/-/api-0.19.0-rc.2.tgz#523def224bfe80e3784890db173bdf49e8dbdec5"
-  integrity sha512-cAJPyjaRGlE+2nmzNdN/o43IHUNGXA4ufMVRZlKGUctxmxhVH4Wfi0xXYj3AhOoWJWlK/KrbdEiEDzmemYX5Fw==
+"@nitric/api@v0.20.0-rc.2":
```

Comment:
Is the details contract released? Can this be changed to a stable release?

Reply:
It's been release to develop, the main thing to watch out for is avoiding production releases with an RC core library.